src/main/java/com/askimed/nf/test/core/ITestExecutionListener.java

```diff
@@ -1,10 +1,13 @@
 package com.askimed.nf.test.core;
 
+import java.io.IOException;
+import javax.xml.stream.XMLStreamException;
+
 public interface ITestExecutionListener {
 
 	public void testPlanExecutionStarted();
 
-	public void testPlanExecutionFinished();
+	public void testPlanExecutionFinished() throws IOException, XMLStreamException;
```

Comment:
`throws Exception` to avoid reporter or file format specific exception?

Reply:
Good idea - we have changed it to Exception now